problemsets-mat223/modules/module3-exercises.tex

```diff
@@ -96,7 +150,42 @@
 				be expressed as spans, express as a translated
 				span.
 		\end{enumerate}
-
+		\begin{solution}
+			\begin{enumerate}
+				\item
+					\begin{enumerate}
+					    \item $\Span\Set*{\mat{0\\-1}}$
+					    \item $\Span\Set*{\mat{3\\-2}}$
+					    \item $\Span\Set*{\mat{4\\5}}$
+					    \item Not possible since $\mat{0\\0}\notin \Set*{\mat{x\\y}\in \R^2: -x-y=-1}$
+					    \item Not possible since $\mat{0\\0}$ is not on the line
+					\end{enumerate}
+				\item
+				%% I don't know how you want these to be labelled. Need more feedback
```

Comment:
Add a `\label{...}` to the appropriate `\item` and then use a `\ref{...}` to reference them.